nexus_constructor/model/load_from_json.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+
+from PySide2.QtWidgets import QWidget
+
+from nexus_constructor.component.component_type import COMPONENT_TYPES
+from nexus_constructor.model.component import Component
+from nexus_constructor.model.entry import Entry
+from nexus_constructor.model.instrument import Instrument
+from nexus_constructor.ui_utils import show_warning_dialog
+
+IGNORE = ["entry", "instrument", "transformations", "NX_class"]
+
+
+def _read_nx_class(entry: list) -> str:
+    for item in entry:
+        if item.get("name") == "NX_class":
+            return item.get("values")
+        if item.get("NX_class"):
+            return item.get("NX_class")
+
+    return ""
+
+
+def _read_transformations(entry: list):
+    for item in entry:
+        if item.get("name") == "transformations":
+            return item.get("children")
+
+    return None
+
+
+def _retrieve_children_list(json_dict: dict) -> list:
+    try:
+        return json_dict["nexus_structure"]["children"][0]["children"]
```

Comment:
I think this makes it a little clearer:
```suggestion
        entry = json_dict["nexus_structure"]["children"][0]
        return entry["children"]
```